evalml/tests/conftest.py

```diff
@@ -2226,6 +2241,102 @@ def imputer_test_data():
     return X
 
 
+@pytest.fixture
+def nullable_type_test_data():
```

Comment:
100% agree! My plan is to have it be configurable similar to the `has_nans` param to decide if users want `Age` to be present (since that, and a small change to call `cool col with nan` `cool with nan` to match the format of the other nan cols, are the only differences between the two fixtures.

Reply:
Reused available info between the imputer fixture and this one: 5b34fc851